packages/react-percy-config/src/normalize.js

```diff
@@ -1,9 +1,11 @@
 import defaults from './defaults';
 import path from 'path';
 
-export default function normalize(config, packageRoot) {
+export default function normalize(config, packageRoot, debug = false) {
```

Comment:
I don't 100% have this flow in my head, but why not add `config.debug` instead of a new param here?

Reply:
Could've done it either way really. Basically this function takes in whatever config is in the user's package.json as well as the debug flag and constructs a clean, formatted object with the Percy config options we use internally. Felt a little cleaner to pass that debug arg in and append it to the cleaned object here rather than having anything outside this function modify the config object.

Reply:
Ah I see, makes sense.